static/chats.js

```diff
@@ -162,6 +168,38 @@ function uzstadiVaardu(jaunaisVards) {
   return `${vecaisVards} kļuva par ${vards}`
 }
 
+function gStavoklis(teksts) {
+  let vardi = teksts.split(" ");
+  /* let izvads = vardi.slice(1); */
+ let izvads = Object.assign({}, vardi.slice(1));
+  
+  fetch('/chats/set_mood', {
+    method: 'POST',
+    headers: {
+        'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({ "mood": izvads })
+});    
+
+gStavoklisNoFaila();
```

Comment:
Domāju, ka šis mums nav nepieciešams. Mēs jau zinām savu personīgo garastāvokli.
Šeit vajag padomāt, kā formatējot rindu varam parādīt lietotāja garastāvokli, lai citi to arī redzētu.

Tāpēc piedāvāju sūtīt uz serveri šādi:
```js
 { lietotajs: "Lietotajs", garastavoklis: "Garastavoklis" }
```
Tad servera pusē saglabāsim failā, kur mums stāvēs lietotāja garastāvoklis, bet pirms rakstīt failā vajadzētu pārliecināties, ka dotajam lietotājam jau nav garastāvokļa. Ja tāds jau ir, tad pārrakstām, ja nav, tad rakstām failā.
Kas notiek, ja lietotājs nomaina savu vārdu? Vajadzētu padomāt, vai varam pirms vārda maiņas nodzēst iepriekšējo garastāvokli.